tests/test_model_operators.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+from base64 import b64decode
+
+import bpy
+
+from mmd_tools.core.model import Model
+
+# Usage: blender --background -noaudio --python tests/test_model_operators.py -- --verbose
+
+# Due to the limitations of Windows this file can't contain characters
+# outside the cp1252 range. Because Blender parses the file with that character set.
+# http://stackoverflow.com/questions/3284827
+# https://developer.blender.org/T35176
+
```

Comment:
Understood. It's kind of a limitation..., we may imporve this in the future if there is a way to solve it. :)